### PR TITLE
10172 move all incubator modules of elico8 to public GitHub connector dns dnspod

### PR DESCRIPTION
**Pull request by Rona111**

### CODE REVIEW

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn

File: connector_dns_dnspod/readme.rst [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Connector Dns Dnspod
+=====================
+
+This module aims to allows you to manage your DNSPod domain through Odoo.
```
Comment (elicoidal) [inline connector_dns_dnspod/readme.rst]: This module allows you to manage your DNSPod domain through Odoo.
It uses the service from www.dnspod.cn and synchronizes (create/delete/update)
all relevant information such as domain names and records.
This module is based on the odoo-connector tehcnology.

File: connector_dns_dnspod/readme.rst [hunk 2]
```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Connector Dns Dnspod
```
Comment (elicoidal) [inline connector_dns_dnspod/readme.rst]: Connector DNS for dnspod.cn

File: connector_dns_dnspod/readme.rst [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+This module aims to allows you to manage your DNSPod domain through Odoo.
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (connector_dns)
+
```
Comment (elicoidal) [inline connector_dns_dnspod/readme.rst]: - odoo-connector framework

File: connector_dns_dnspod/security/ir.model.access.csv [hunk 4]
```diff
@@ -0,0 +1,10 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+
```
Comment (elicoidal) [inline connector_dns_dnspod/security/ir.model.access.csv]: remove empty lines

File: connector_dns_dnspod/static/description/index.html [hunk 5]
```diff
@@ -0,0 +1,111 @@
+<section class="oe_container oe_dark">
```
Comment (elicoidal) [inline connector_dns_dnspod/static/description/index.html]: Do we still need this file?

**Comment by Rona111** [comment]
cc @elicoidal @seb-elico @duanyp1991 @siyuancn  the photo's url is base to the github branch,if it's ok,I should change the url to the elico branch .

**Comment by elicoidal** [comment]
LGTM

**Comment by Rona111** [comment]
@elicoidal  already change the photo url to elico.

**Comment by elicoidal** [comment]
I think this is not correct. It should nonot make reference to github but to the directory in the module.
@siyuancn @duanyp1991 can you recheck that?

**Comment by siyuancn** [comment]
It seems we need host images on web and specify the image path in README
@Rona111 can you check the link and try wether there is a better solution:
http://stackoverflow.com/questions/14494747/add-images-to-readme-md-on-github

**Comment by elicoidal** [comment]
Can't we use a simple local path?
It wont display in github but it should display properly otherwise

**Comment by siyuancn** [comment]
I try in my local and it works with company logo: logo.png
etc: .. image:: logo.png
But it didn't work with the local path like:
.. image:: {module_name}/path/picture.png

I don't have time to do the further research.

@Rona111 Can you try to find how it works with the company log file: logo.png?

**Comment by elicoidal** [comment]
duplicated @noahzaozao

**Pull request closed without merging**